supports/tools/docker/README.md

```diff
@@ -1,8 +1,53 @@
+# Run Hadoop cluster with SSM in docker containers
+
+There are two cluster types:
+* singlehost
+* multihost
+
+## Singlehost configuration
+
+* Hadoop + SSM in one container
+* SSM metastore as postgres container
+
+Command to build docker images in singlehost cluster mode (from project root dir)
+
+```shell
+build-image.sh singlehost
+```
+
+Command to start docker containers 
+
+```shell
+docker-compose -f ${project-dir}/supports/tools/docker/singlehost/docker-compose.yaml up -d
```

Comment:
The web interface of the SmartServer is not accessible.
![image](https://github.com/user-attachments/assets/3a699690-60eb-4696-8352-15d0294a9dd2)

```shell
root@4e7d8f4903de:/# curl http://localhost:8081/
{"timestamp":"2024-08-16T07:08:58.230+00:00","status":404,"error":"Not Found","path":"/"}
```

Reply:
fixed